frontend/apps/web/components/jobs/SchemaTable/FormErrorsCard.tsx

```diff
@@ -35,8 +35,8 @@ export default function FormErrorsCard(props: Props): ReactElement {
   const messages = formErrorsToMessages(formErrors);
   return (
     <Card className="w-full flex flex-col">
-      <CardHeader className="flex flex-col gap-2">
-        <div className="flex flex-row items-center justify-between">
+      <CardHeader className="flex flex-col">
+        <div className="flex flex-row items-center justify-between h-8">
```

Comment:
I tested a few forms that use this component on staging and production and they overall look fine to me in that when they render, they don't adjust the components at all (which that other PR seems to be changing?). So why the change here?

Also, this form errors card still uses the simpler form error syntax that just renders the text as red.
Did the changes to the `<FormMessage />` component bring on the need to adjust the height? Or was that an issue with the original shadcn component styling too?

Reply:
it's an issue with the original component before any of my <FormMessage /> updates. The <FormMessage /> updates just reveled it. 

The original row without with badge is 16px tall.
![image](https://github.com/user-attachments/assets/72d9d01c-01b2-4818-be2c-e7f0092d4be8)

When the badge gets added in, it jumps to 22px
![image](https://github.com/user-attachments/assets/75c35e27-d375-4b42-b074-be97ebd53646)

That 6px causes things to jump down 6px to make room for it. 

So with this change:

We set the row height to 32px so that it can accommodate the badge:
![image](https://github.com/user-attachments/assets/62d14080-db55-482f-bf50-472c0c957f47)

https://www.loom.com/share/7fee6c5a9839412e87330682cec5046d